### DISCUSSION TITLE
[@types/node]: Blob constructor may need to be updated

### DISCUSSION THREAD

**Discussion by peidongxie** [DefinitelyTyped/DefinitelyTyped#61436] [Issues with a @types package]
[new buffer.Blob([sources[, options]])](https://nodejs.org/api/buffer.html#new-bufferblobsources-options)
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/8e17f9ca957a06040badb53ae7688fbb74229ccf/types/node/buffer.d.ts#L137

1. The type of `sources`
- expect: `string[] | ArrayBuffer[] | TypedArray[] | DataView[] | Blob[]`
- actual: `string[] | TypedArray[] | DataView[] | Blob[]`

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/8e17f9ca957a06040badb53ae7688fbb74229ccf/types/node/crypto.d.ts#L677

2. The type of `options`
- expect: `{ endings?: 'transparent' | 'native' | undefined; type?: string | undefined; }`
- actual: `{ encoding?: BufferEncoding | undefined; type?: string | undefined; }`

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/8e17f9ca957a06040badb53ae7688fbb74229ccf/types/node/buffer.d.ts#L101-L112

[a comment by an automated account is not shown]

**Comment by ghost** [comment]
Fixed in PR #68470
